react/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -195,7 +195,9 @@ function CreateShipmentView() {
                 message: "Successfully created a new shipment",
               });
 
-              navigate(`/bases/${baseId}/transfers/shipments`);
+              navigate(
+                `/bases/${baseId}/transfers/shipments/${mutationResult.data?.createShipment?.id}`,
```

Comment:
maybe defining a variable for the shipment ID makes sense because the same expression is used in line 193